core/src/org/pentaho/di/core/sql/SQLUtil.java

```diff
@@ -48,14 +38,17 @@
 import org.pentaho.di.core.row.value.ValueMetaString;
 import org.pentaho.di.core.xml.XMLHandler;
 
-/**
- * Static methods provided by this class should be copied to their respective projects
- *
- * Data Service client code is now available in the pdi-dataservice-plugin project
- *
- */
-@Deprecated
-public class ThinUtil {
+import java.math.BigDecimal;
```

Comment:
What's the use in kettle-core?  Is it just for other stuff that's only used by dataservices?